### ISSUE TITLE
7.2.11 AdoptOpenJDK with Edge browser intergration seems broken

### ISSUE THREAD

**Issue by nahoijtink** [subhra74/xdm#404]
I gave a fairly clean install of Windows 10:

Edition	Windows 10 Home
Version	20H2
Installed on	‎14/‎11/‎2020
OS build	19042.630
Experience	Windows Feature Experience Pack 120.2212.31.0

All Windows updates are applied. 

I can use the 7.2.10 (windows .msi install) with the Edge plugin perfectly fine following the steps described in the install instructions. 

However the 7.2.11 (windows .msi installer) version does not connect to the browser plugin 1.4. Either the plugin does not connect at all or the plugin does not properly communicate with XDM. I can't tell which is the case, It will not always have a red cross on it from the start, but clicking on it will turn it red either case.

I then installed JRE 8 and JDK 15. Downloaded the 7.2.11 Portable JAR, and that works fine with the Edge XDM browser plugin 1.4.

I exited the portable version and the started the .MSI installed executable and now that seems to work fine with the Edge XDM browser plugin 1.4.

In conclusion, the included Java AdoptOpenJDK 11.0.6 and XDM 7.2.11 have some sort of problem communicating with the Edge Plugin 1.4 when no 'official' Java support is installed on Windows 10.

**Comment by subhra74** [comment]
Please use the latest Chromium based Edge and install the chrome plugin

**Comment by nahoijtink** [comment]
Hi,


It was the latest version of Edge (Chromium based). In fact, the Windows 10 is also the latest version (20H2). The OS, Edge, and everything else is fully upated to the latest versions. The PC was cleanly installed when i installed and tested XDM. I have XDM running, so that is not the issue. The issue is that out-of-the-box the XDM Plugin 1.4/2.1 does not communicate with XDM on Edge Chromium when using the official installer that installs AdoptOpenJDK .

Now, i don't mind personally, i can and did install Java as mentioned in my submitted bug report, but for other less experienced users this might not be the case. They trust an installer to deliver a product which runs out-of-the-box. I hope we can agree to that, and hopefully you can resolve the issue as mentioned in the bug report. You might consider using a VM to do testing on a clean Windows 10 20H2 with the latest Edge Chromium browser to verify my experience.


________________________________
Van: Subhra Das Gupta <notifications@github.com>
Verzonden: zaterdag 12 december 2020 21:52
Aan: subhra74/xdm <xdm@noreply.github.com>
CC: nahoijtink <elkino@live.nl>; Author <author@noreply.github.com>
Onderwerp: Re: [subhra74/xdm] 7.2.11 AdoptOpenJDK with Edge browser intergration seems broken (#404)


Please use the latest Chromium based Edge and install the chrome plugin
